android/app/src/main/java/app/organicmaps/util/SharingUtils.java

```diff
@@ -92,7 +103,28 @@ public static void shareBookmark(@NonNull Context context, @NonNull BookmarkInfo
     context.startActivity(Intent.createChooser(intent, context.getString(R.string.share)));
   }
 
-  public static void shareBookmarkFile(Context context, String fileName)
+  public static ActivityResultLauncher<Intent> RegisterLauncher(@NonNull Fragment fragment)
+  {
+    return fragment.registerForActivityResult(
+            new ActivityResultContracts.StartActivityForResult(), result -> {
```

Comment:
How this indent was calculated? :) Why 8 spaces instead of 4? Or like below?

```suggestion
    return fragment.registerForActivityResult(new ActivityResultContracts.StartActivityForResult(), result ->
    {
```

Reply:
it's all IDE :) fixed